recipe/thirdparty/emaildelivery/smtpService/evServiceImplementation.go

```diff
@@ -0,0 +1,23 @@
+package smtpService
```

Comment:
You do not need this type as the emails sent from third party are exactly the same as those send from email verification recipe